1-General-Overview-of-the-System.md

```diff
@@ -71,4 +71,4 @@ The kernel sometimes wants to prevent occurrence of interrupts when it is doing
 
 ### Memory Management
 
-The kernel permanently resides in the main memory. When compiling a program, the compiler generates a set of addresses in the program that represent addresses of variables and data structures of the addresses of instructions such as functions. The compiler generates the addresses for a *virtual machine* as if no other program will execute simultaneously on the physical machine. When the program is run on the machine, the kernel allocates space in main memory for it, but the virtual addresses generated by the compiler need not be identical to the physical addresses that they occupy in the machine. The kernel coordinates with the machine hardware to set up a virtual to physical address translation that maps the compiler-generated addresses to the physical machine addresses.
```

Comment:
I don't see any change here. Maybe only the line ending has changed. Will you please remove this change and resubmit? Thanks @omkar98 for your help! :)

Reply:
Oh! Yes. There's no change here. I did that by mistake..